spinnman/messages/eieio/create_eieio_data.py

```diff
@@ -118,145 +118,146 @@
 
```

Comment:
This whole class seems wholely more complex than it needs to be, and feels very unscalable and unmanageable. To top it off, it seems it is missing certain classes of EIEIO data messages. There must be a cleaner way to interprete the EIEIO data packet interface. Though because this is rather tied into the buffered in functionality (and likely the buffered out stuff), im going to leave it for a future refactor. but OWCH!